plugins/optimization-detective/class-od-tag-visitor-context.php

```diff
@@ -16,6 +16,8 @@
  *
  * @since 0.4.0
  * @access private
+ *
+ * @property-read OD_URL_Metric_Group_Collection $url_metrics_group_collection
```

Comment:
I don't think it's worth adding this, given it's only for backward compatibility.

Reply:
It's needed for static analysis. In 5960ff3c I've added a note that it is deprecated.